.env.development.example

```
@@ -37,7 +37,7 @@ AUTH_GITHUB_AUTHORIZE_URL=https://github.com/login/oauth/authorize
 AUTH_GITHUB_ACCESS_URL=https://github.com/login/oauth/access_token
 AUTH_GITHUB_PROFILE_URL=https://api.github.com/user
 AUTH_GITHUB_ALLOW_PUBLIC_REGISTRATION=true
-AUTH_GITHUB_SCOPE=read:user,user:email
+AUTH_GITHUB_SCOPE=read:user,user:email,read:org
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The AUTH_GITHUB_SCOPE key should go before the AUTH_PROVIDERS key

(UnorderedKey)